dockerfile

```diff
@@ -1,7 +1,37 @@
-# Configure nwserver to run with nwnx
+# Load nwnx image to import nwserver + nwnx plugins
 ARG NWNX_VERSION
-FROM nwnxee/unified:${NWNX_VERSION}
+FROM nwnxee/unified:${NWNX_VERSION} as nwnx
+
+# Remove incompatible plugins
+RUN rm -rf /nwn/nwnx/NWNX_Ruby.so \
+    /nwn/nwnx/NWNX_SpellChecker.so \
+    /nwn/nwnx/NWNX_Redis.so
```

Comment:
Hi @jhett12321, do you remember what this incompatibility was, in particular for `NWNX_Redis`? I have nss that uses `NWNX_Redis` and I'm wondering if I _have_ to migrate that logic to Anvil in order to use this image, or if I can simply re-introduce the plugin.